Example/GiniVision/ComponentAPICoordinator.swift

```diff
@@ -135,7 +135,7 @@ final class ComponentAPICoordinator: NSObject, Coordinator {
     }
     
     fileprivate func showResultsTableScreen(forDocument document: GINIDocument,
-                                            withResult result: GINIResult) {
+                                            withResult result: [String: GINIExtraction]) {
```

Comment:
Why did you decide to remove the `GINIResult` typedef?

Reply:
I consider it more confusing, since most of the times I have to check what kind of dict it is.

Reply:
Yeah, I see your point.

One alternative is always to make a class/struct that encapsulates that information. It is probably an overkill for this case, but you might consider it if you feel confused while using these methods.

Reply:
For me it is more clear this way, since make it more raveled could make it more confusing for others.